diff --git a/node_modules/svg2ttf/index.js b/node_modules/svg2ttf/index.js
index 762a1db..8cadfe3 100644
--- a/node_modules/svg2ttf/index.js
+++ b/node_modules/svg2ttf/index.js
@@ -14,7 +14,7 @@ var svg     = require('./lib/svg');
 var sfnt    = require('./lib/sfnt');
 
 
-var VERSION_RE = /^(Version )?(\d+[.]\d+)$/i;
+var VERSION_RE = /^(Version )?((\d+[.]\d+)[\w.,; ]*)$/i;
 
 
 function svg2ttf(svgString, options) {
@@ -40,7 +40,16 @@ function svg2ttf(svgString, options) {
     throw new Error('svg2ttf: invalid option, version - "' + options.version + '"');
   }
 
-  versionString = 'Version ' + versionString.match(VERSION_RE)[2];
+  var fontVersion = versionString.match(VERSION_RE);
+
+  // Like SourceSansPro's Version 1.095;hotconv 1.0.109;makeotfexe 2.5.65596
+  // or 10.006 beta1
+  versionString = 'Version ' + fontVersion[2];
+
+  // We must update the fontRevision field and ensure that a specific string is
+  // supported. See also: https://silnrsi.github.io/FDBP/en-US/Versioning.html
+  font.revision = fontVersion[3]; // Only number
+
   font.sfntNames.push({ id: 5, value: versionString }); // version ID for TTF name table
   font.sfntNames.push({ id: 6, value: (options.fullname || svgFont.id).replace(/[\s\(\)\[\]<>%\/]/g, '').substr(0, 62) }); // Postscript name for the font, required for OSX Font Book
 
